tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+This configuration allows the system to withstand failures in one zone by replicating resources and workloads across multiple zones.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a domain managed on the Scaleway [Domains and DNS service](/network/domains-and-dns/quickstart/).
+  - You have installed [Terraform](/tutorials/terraform-quickstart/) on your local computer
+</Message>
+
+
+## Deploying the Kapsule Kubernetes cluster
+
+Start by creating a multi-AZ cluster on `fr-par` region, in a dedicated VPC and Private Network, with three pools of nodes, one in `fr-par-1`, one in `fr-par-2` and one in `fr-par-3`, with two nodes each.
+
+<Lightbox src="scaleway-k8s-kapsule-multi-az-1.webp" alt="" size="large" />
+
+1. Create a `main.tf` file that will contain the Terraform part of this project, and set the required providers and their versions
+    ```
+    terraform {
+        required_providers {
+            scaleway = {
+                source  = "scaleway/scaleway"
+                version = ">= 2.28.0"
+            }
+            kubernetes = {
+                source  = "hashicorp/kubernetes"
+                version = ">=2.23.0"
+            }
+            helm = {
+                source  = "hashicorp/helm"
+                version = "2.11.0"
+            }
+            local = {
+                source  = "hashicorp/local"
+                version = "2.4.0"
+            }
+            kubectl = {
+                source  = "gavinbunney/kubectl"
+                version = "1.14.0"
+            }
+        }
+        required_version = ">= 1.0.0"
+    }
+    ```
+
+2. Add the Scaleway provider and its configuration. Refer to the Scaleway Terraform provider documentation for more information on how to [configure the provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs).
+    ```
+    provider "scaleway" {
+        region  = "fr-par"
+    }
+    ```
+
+3. Now create a dedicated VPC and a Private Network where we will deploy the Kapsule cluster. VPC and Private Networks are region-wide, so you can use them to host a cluster spread on multiple availability zones.
+    ```
+    resource "scaleway_vpc" "vpc_multi_az" {
+    name = "vpc-multi-az"
+    tags = ["multi-az"]
+    }
+
+    resource "scaleway_vpc_private_network" "pn_multi_az" {
+    name   = "pn-multi-az"
+    vpc_id = scaleway_vpc.vpc_multi_az.id
+    tags   = ["multi-az"]
+    }
+    ```
+
+4. Add the Kapsule cluster and the three pools, each in a different availability zone but in the same Private Network, to effectively make the cluster multi-AZ.
+Refer to the Scaleway Terraform provider documentation for available [Kapsule parameter]([https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_cluster), there is no specific configuration for the multi-AZ mode on the `scaleway_k8s_cluster`.
```

Comment:
- Kapsule parameters (plural)?
- I suggest to start a new phrase on the second part: 
There is no specific configuration for the multi-AZ mode on the `scaleway_k8s_cluster`.
Even better, I suggest to start a bullet list, so that cluster then pool is discussed sequentially. 
Rephrased like:

- `scaleway_k8s_cluster` requires no specific configuration for the multi-AZ mode.
- The [pools configuration](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_pool) `scaleway_k8s_pool` will get a `zone` parameter that allows placing each one in a different availability zone.

Reply:
```suggestion
Refer to the Scaleway Terraform provider documentation for available [Kapsule parameters]([https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_cluster).
- `scaleway_k8s_cluster` requires no specific configuration for the multi-AZ mode.
- The [pools configuration](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_pool) `scaleway_k8s_pool` will get a `zone` parameter that allows placing each one in a different availability zone.
```

Reply:
There's an extra `[` at the beginning of the link too